Update Readme and changelog

Andre601 opened the pull request:
Thank you for contributing! Please check the following things before submiting your PR:

Required:
- [x] I have read and followed the [contributing guidelines](https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md)

If necessary:
- [x] I have updated the README and documentation.
- [x] I have updated the ChangeLog with the changes I have made.

> Updated README and ChangeLog

<!--
If this pull request closes an issue, then add the issue number below
-->

Closes #

---------------------------------------------------------------------

**Note:** we will close your PR without comment if you do not check the required boxes above and provide ALL requested information.

Andre601 commented:
Moved the examples to a [Wiki page](https://github.com/Readme-Workflows/recent-activity/wiki/Examples) as I feel it becomes kinda bloated now.

abhijoshi2k commented:
@Andre601 The next version won't be 1.2.0 
So Please make it 1.2.3 in readme.

abhijoshi2k commented:
> > @Andre601 The next version won't be 1.2.0
> > So Please make it 1.2.3 in readme.
> 
> Why not 1.2.1 ?

I mistyped. Sorry for that.
We'll go with 1.2.1

Andre601 commented:
Soooooooo.... Updated changelog file and renamed it to `CHANGELOG.md` to go with common practices.

I'll update the release date of v1.2.1 once it's out.

abhijoshi2k commented:
Okay then. Merging this PR.
